### DISCUSSION TITLE
Option to add sql comments to generated SQL for SlowLog and OpenTracing

### DISCUSSION THREAD

**Discussion by adamhawkinselt** [mikro-orm/mikro-orm#3764] [Ideas]
**Is your feature request related to a problem? Please describe.**
We use Elastic APM as our OpenTracing client which allows us to see the SQL being executed. Frequently we are wrapping this in manual tracing spans to understand the purpose of the SQL, Also the query statistics from the DB Engine e.g. Slow log, will reference the SQL though this is very hard to link back to the code. Allowing addition of comments to be added the generated SQL would allow description or trace id to be added to the SQL which would be visible in the OpenTracing / Logging / DB tooling.

**Describe the solution you'd like**
Additional attribute added to xOptions class to allow a comment to be passed which would be prefixed to the generated SQL.

**Describe alternatives you've considered**
Can't see any way around this, should have a negligible performance impact. 

**Additional context**

**Comment by B4nan** [comment]
You will need to be more specific and provide some examples, e.g. options of what?
